### PR TITLE
Add stock_inventory_line_domain

### PR DESCRIPTION
**Pull request by ddufresne**
https://www.numigi.com/web#id=18759&action=298&model=project.task&view_type=form&menu_id=200

### CODE REVIEW

**Comment by ddufresne** [comment]
depends on https://github.com/Numigi/odoo-public/pull/62

File: stock_inventory_line_domain_barcode/models/stock_inventory.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# © 2020 - today Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import models
+from odoo.exceptions import ValidationError
```
Comment (ddufresne) [inline stock_inventory_line_domain_barcode/models/stock_inventory.py]: Codacy Issue found: [Unused ValidationError imported from odoo.exceptions](https://app.codacy.com/manual/numigi/odoo-stock-addons/pullRequest?prid=4894269)

File: stock_inventory_line_domain/models/__init__.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# © 2020 - today Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from . import (
```
Comment (ddufresne) [inline stock_inventory_line_domain/models/__init__.py]: Codacy Issue found: ['.stock_inventory' imported but unused](https://app.codacy.com/manual/numigi/odoo-stock-addons/pullRequest?prid=4894269)

File: stock_inventory_line_domain/models/stock_inventory.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
+# © 2020 - today Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models, _
```
Comment (ddufresne) [inline stock_inventory_line_domain/models/stock_inventory.py]: Codacy Issue found: [Unused fields imported from odoo](https://app.codacy.com/manual/numigi/odoo-stock-addons/pullRequest?prid=4894269)

Comment (ddufresne) [inline stock_inventory_line_domain/models/stock_inventory.py]: Codacy Issue found: ['odoo.fields' imported but unused](https://app.codacy.com/manual/numigi/odoo-stock-addons/pullRequest?prid=4894269)

File: stock_inventory_line_domain_barcode/models/__init__.py [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# © 2020 - today Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from . import (
```
Comment (ddufresne) [inline stock_inventory_line_domain_barcode/models/__init__.py]: Codacy Issue found: ['.stock_inventory' imported but unused](https://app.codacy.com/manual/numigi/odoo-stock-addons/pullRequest?prid=4894269)

File: stock_inventory_line_domain_barcode/models/stock_inventory.py (hunk 1 above)
Comment (ddufresne) [inline stock_inventory_line_domain_barcode/models/stock_inventory.py]: Codacy Issue found: ['odoo.exceptions.ValidationError' imported but unused](https://app.codacy.com/manual/numigi/odoo-stock-addons/pullRequest?prid=4894269)

Comment (ddufresne) [inline stock_inventory_line_domain_barcode/models/stock_inventory.py]: Codacy Issue found: ['odoo.exceptions.ValidationError' imported but unused](https://app.codacy.com/manual/numigi/odoo-stock-addons/pullRequest?prid=4894269)

Comment (ddufresne) [inline stock_inventory_line_domain_barcode/models/stock_inventory.py]: Codacy Issue found: [Unused ValidationError imported from odoo.exceptions](https://app.codacy.com/manual/numigi/odoo-stock-addons/pullRequest?prid=4894269)

**Review by foutoucour — APPROVED** [review]

**Pull request merged**